cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -78,7 +77,8 @@ function Autocomplete(element, opts) {
   };
 
   // Search variables
-  let _xhr;
+  const searchRequest = new AbortController();
+  const { signal } = searchRequest.signal;
```

Comment:
This needs to be managed slightly differently. Right now all the `fetch`s are reusing the same `signal` from this one `AbortController` whereas we need to make a new `AbortController` per `fetch` (and abort the last one).

Right now, if a request is aborted, all subsequent requests will fail.

Reply:
Did I do this right? https://github.com/cfpb/consumerfinance.gov/pull/7384/commits/f54c906b9c01fc00aa6993d5968930972cac6de0